docs/vta/install.md

```diff
@@ -20,8 +20,9 @@ VTA Installation Guide
 
 We present three installation guides, each extending on the previous one:
 1. [Simulator installation](#vta-simulator-installation)
-2. [Hardware test setup](#vta-pynq-based-test-setup)
-3. [FPGA toolchain installation](#vta-fpga-toolchain-installation)
+2. [PYNQ-based test setup](#vta-pynq-based-test-setup)
```

Comment:
Perhaps we can rename these categories to "FPGA test setup" and have sub categories for "Pynq board", "DE10Nano board" etc.?